apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -39,6 +44,34 @@ export const getPageById = (id: number) => {
     .executeTakeFirstOrThrow()
 }
 
+export const updatePageById = (
+  page: Partial<Omit<Page, 'id' | 'blobId'>> & { id: string },
+) => {
+  const { id, ...rest } = page
+  return db.transaction().execute((tx) => {
```

Comment:
Just curious, is there a reaction why we wrap what appears to be a single sql statement in a transaction?

Reply:
defensive programming here - idt the `tx` will do anything different (apart from db sql actually calling `BEGIN TRANSACTION`) but i think having here helps us to avoid potential errors down the line so hapy to keep it